main.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative './Models/author'
+require_relative './Models/book'
+require_relative './Models/order'
+require_relative './Models/reader'
+require_relative './Models/library'
+require_relative 'seeds'
+
+library = Library.new([], [], [], [])
```

Comment:
What is `[], [], [], []`? Use named arguments instead or hash. Same about all other models

Reply:
fixed